src/Configuration/Configuration.php

```diff
@@ -115,7 +116,8 @@ public function __construct(
         ?float $minCoveredMsi,
         int $msiPrecision,
         int $threadCount,
-        bool $dryRun
+        bool $dryRun,
+        array $preFilters
```

Comment:
What's the type of this parameter? According to the schema, this looks like `@psalm-param list<string>`

Reply:
It is. It's going to be checked by PHPStan downstream.

Reply:
There's no type declaration for property, parameter or getter, hence my question. 

I later decyphered it by looking at the tests, but overall I endorse declaring this explicitly. See my suggestion to require a `list<class-string<NameOftThePluginInterfaceHere>>`, so that the interface can declare an explicit zero-argument named constructor or getter.